subsys/nrf_security/src/drivers/Kconfig.psa_accel

```diff
@@ -0,0 +1,821 @@
+	bool
+
+config PSA_ACCEL_RSA3072_SHA224_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA3072_SHA224_PSS
+	bool
+
+config PSA_ACCEL_RSA3072_SHA256_OAEP
+	bool
+
+config PSA_ACCEL_RSA3072_SHA256_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA3072_SHA256_PSS
+	bool
+
+config PSA_ACCEL_RSA3072_SHA384_OAEP
+	bool
+
+config PSA_ACCEL_RSA3072_SHA384_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA3072_SHA384_PSS
+	bool
+
+config PSA_ACCEL_RSA3072_SHA512_OAEP
+	bool
+
+config PSA_ACCEL_RSA3072_SHA512_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA3072_SHA512_PSS
+	bool
+
+config PSA_ACCEL_RSA4096_PKCS1V15_CRYPT
+	bool
+
+config PSA_ACCEL_RSA4096_SHA1_OAEP
+	bool
+
+config PSA_ACCEL_RSA4096_SHA1_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA4096_SHA1_PSS
+	bool
+
+config PSA_ACCEL_RSA4096_SHA224_OAEP
+	bool
+
+config PSA_ACCEL_RSA4096_SHA224_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA4096_SHA224_PSS
+	bool
+
+config PSA_ACCEL_RSA4096_SHA256_OAEP
+	bool
+
+config PSA_ACCEL_RSA4096_SHA256_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA4096_SHA256_PSS
+	bool
+
+config PSA_ACCEL_RSA4096_SHA384_OAEP
+	bool
+
+config PSA_ACCEL_RSA4096_SHA384_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA4096_SHA384_PSS
+	bool
+
+config PSA_ACCEL_RSA4096_SHA512_OAEP
+	bool
+
+config PSA_ACCEL_RSA4096_SHA512_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA4096_SHA512_PSS
+	bool
+
+config PSA_ACCEL_RSA6144_PKCS1V15_CRYPT
+	bool
+
+config PSA_ACCEL_RSA6144_SHA1_OAEP
+	bool
+
+config PSA_ACCEL_RSA6144_SHA1_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA6144_SHA1_PSS
+	bool
+
+config PSA_ACCEL_RSA6144_SHA224_OAEP
+	bool
+
+config PSA_ACCEL_RSA6144_SHA224_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA6144_SHA224_PSS
+	bool
+
+config PSA_ACCEL_RSA6144_SHA256_OAEP
+	bool
+
+config PSA_ACCEL_RSA6144_SHA256_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA6144_SHA256_PSS
+	bool
+
+config PSA_ACCEL_RSA6144_SHA384_OAEP
+	bool
+
+config PSA_ACCEL_RSA6144_SHA384_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA6144_SHA384_PSS
+	bool
+
+config PSA_ACCEL_RSA6144_SHA512_OAEP
+	bool
+
+config PSA_ACCEL_RSA6144_SHA512_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA6144_SHA512_PSS
+	bool
+
+config PSA_ACCEL_RSA8192_PKCS1V15_CRYPT
+	bool
+
+config PSA_ACCEL_RSA8192_SHA1_PSS
+	bool
+
+config PSA_ACCEL_RSA8192_SHA1_OAEP
+	bool
+
+config PSA_ACCEL_RSA8192_SHA1_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA8192_SHA224_OAEP
+	bool
+
+config PSA_ACCEL_RSA8192_SHA224_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA8192_SHA224_PSS
+	bool
+
+config PSA_ACCEL_RSA8192_SHA256_OAEP
+	bool
+
+config PSA_ACCEL_RSA8192_SHA256_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA8192_SHA256_PSS
+	bool
+
+config PSA_ACCEL_RSA8192_SHA384_OAEP
+	bool
+
+config PSA_ACCEL_RSA8192_SHA384_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA8192_SHA384_PSS
+	bool
+
+config PSA_ACCEL_RSA8192_SHA512_OAEP
+	bool
+
+config PSA_ACCEL_RSA8192_SHA512_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA8192_SHA512_PSS
+	bool
+
+config PSA_ACCEL_SHA1
+	bool
+
+config PSA_ACCEL_SHA224
```

Comment:
Please follow naming convention splitting hash-algorithm and size:
https://armmbed.github.io/mbed-crypto/html/api/ops/hashes.html#c.PSA_ALG_SHA_224
(Also followed in PSA_WANT_ALG_SHA_224)

Reply:
Sorry but these come from Oberon PSA so I will not update it, any requested changes we need to do to them. Otherwise we will be out of sync with them and we will have to maintain different options and their matches. I will not do that.